labs/b1.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Beginner Lab 1 - Unix, the Shell, OSS
+layout: lab
+---
+
+Welcome to the first lab! Here are some commands you might find helpful:
+`cat, cd, emacs, file, grep, less, ls, man, nano, tar, wget, vim`
+
+Remember: Google and `man` are your friends!
+
+**Answer these questions on Gradescope**, the assignment should be up already. If you haven't been added to the class yet, let a head facilator know!
+
+### General Questions
+
+First we'll do some general questions to get you comfortable with working in the shell before we get to the real lab assignment. Feel free to use Google and work in a terminal (where applicable) to verify your conjectures.
+
+1. What differentiates Linux/OSX from operating systems like Windows?
+
+2. What are some differences between the command line and normal (graphical) usage of an OS?
+
+3. What is the path to root in linux?
+
+4. How can you use a command on a file not in a folder in your current directory?
+
+5. Give a command that can edit a file in root from anywhere in your file system.
+
+6. What are 1 or 2 options/flags and what do they do?
+
+7. How would you create a copy of a file (multiple possible answers)?
+
+8. Which command deletes?
+
+9. How do you make it delete folders?
+
+10. What command modifies permissions? (multiple choice: chmod, ls, mkdir, nano)
+
+11. Which letter is short for the flag corresponding to the permission to write? (multiple choice: x, w, r)
+
+12. What would the contents of cat.txt be after running "echo 'hello'> cat.txt"
+
+13. What about now after I also run "echo 'world' >> cat.txt"
+
+14. What is the difference between '>' and '>>'?
```

Comment:
There's not enough context in the previous two questions to adequately intuit that `>` (over)writes and `>>` appends, without having another question that has them `echo yes > cat.txt` afterwards.